### PR TITLE
feat: support twenty batch endpoints 

### PR DESCRIPTION
**Pull request by Nabhag8848**
### Description
- Support batch endpoints for twenty 
- Reason for change sdk spec is to match the newest openAPI spec of twenty.

https://github.com/openint-dev/openint/assets/65061890/847d83f0-4974-4c25-ace4-9c61658c1eac

### CODE REVIEW

File: connectors/connector-revert/server.ts [hunk 1]
```diff
@@ -23,36 +23,39 @@ export const revertServer = {
           continue
         }
         const sState = ((state ?? {}) as Record<string, unknown>)[name] ?? {}
-        yield* iterateRecordsInStream(name, stream.fields ?? [], sState) // TODO(@jatin): update this.
+        yield* PipeRecordsInStream(name, stream.fields ?? [], sState) // TODO(@jatin): update this.
```
Comment (basinhr) [inline connectors/connector-revert/server.ts:26]: why the capital function name?

File: connectors/connector-revert/server.ts [hunk 2]
```diff
@@ -23,36 +23,39 @@ export const revertServer = {
...
         })
-        yield res.data.results.map((com) =>
-          helpers._opData(stream as 'company', com.id ?? '', com),
-        )
+
+        records.push(...res.data.results)
         cursor = res.data.next
         if (!cursor) {
+          yield helpers._opData(stream as 'company', '', {
+            [stream]: records,
```
Comment (basinhr) [inline connectors/connector-revert/server.ts:50]: This doesn't quite look right. We need to emit individual records, see example in QBO/server

**Review by basinhr — CHANGES_REQUESTED** [review]

**Review by tonyxiao — CHANGES_REQUESTED** [review]
We do not want to emit arrays of items, but individual items themselves. 
 
 image
Please see `postgres/server.ts` for an example of how to collect individual events into arrays that can be batch inserted.

**Comment by jatinsandilya** [comment]
@tonyxiao seems like the requested changes are made? Good to merge?

**Pull request merged**